next.config.js

```diff
@@ -30,6 +30,19 @@ let nextConfig = {
         permanent: true,
         basePath: false,
       },
+      // REMOVE THIS NEAR LAUCH DATE
+      {
+        source: "/strategies/SteadyETH/manage",
+        destination: "/strategies/SteadyETH",
+        permanent: true,
+        basePath: false,
+      },
+      {
+        source: "/strategies/SteadyBTC/manage",
+        destination: "/strategies/SteadyBTC",
+        permanent: true,
+        basePath: false,
+      },
```

Comment:
this looks like a hacky way, let's do it in `src/pages/strategies/[id]/manage.tsx` you need to filter the `id` that `launchDate` < today